#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
float64 speed
float64 low_speed
float64 full_speed_curvature
float64 low_speed_curvature
float64 deceleration
float64 look_ahead_time
float64 max_look_ahead_distance
float64 min_look_ahead_distance